requirements.dev.txt

```diff
@@ -1,3 +1,5 @@
+-r requirements.txt
+twine
```

Comment:
Could you pin the version of this

Reply:
At the moment twine doesn't need a pinned version since it's only used during the upload step of the CI process.  Using the latest version provided by or pulled from pypi seems the most logical for now.

Reply:
We should pin it to the current major in case the API changes.

Reply:
Looking through their history, it looks like the last change was done in 2018 and they released twine to go with it (and before that almost never).   If they do make api changes, we will get those with the updated twine.  Since this doesn't break anything other than uploading a built package (and not how the package works or is built), I'm inclined to leave it as is until we have a reason to pin it to a version.